views/help/howtos.pug

```diff
@@ -0,0 +1,13 @@
+extends ../layouts/help-layout
+
+block content
+  h1 Getting Started
+
+  .topic-content
+    iframe(width="560" height="315" frameborder="0" allowfullscreen)
+    .content Select a topic
+
+  ul
+    each topic in topics
+      li
+        a.btn.btn-default.topic-selector(href='#', data-id=topic.id, data-content=topic .content, data-url=topics.url )= topics.title
```

Comment:
This line exceeds 80 characters - is there a way in pug to have line continuation?